server/actions/repo/scripts/c_script.sh

```diff
@@ -2,6 +2,18 @@
 #Script for grading C programs (CSE 5)
 #Usage: ./c_script.sh <Optional Bonus Due Date>
 #Example: ./c_script.sh "01/24/2015 15:00"
+
+function appendLinesOver80CharToHTMLFile() {
+    numLinesOverMax=`cat $2 | grep -c '^.\{81,\}'`
+    maxChars=("There are $numLinesOverMax lines over 80 characters\n")
+    if [ $numLinesOverMax -ne 0 ]; then
+        maxChars+="\n"
+        maxChars+=`grep -n '^.\{81,\}' $2`
```

Comment:
Just curious what does this line do? Can we add a comment?

Reply:
line 11: gets the actual lines that are over 80 character long